__tests__/app.test.js

```diff
@@ -39,4 +39,37 @@ describe('/api', ()=>{
             expect(body.endpoints).toEqual(endpoints)
         })
     })
+})
+
+describe('/api/articles/:article_id', ()=>{
+    test('GET:200 responds with a specific article object', ()=>{
+        const article_1 = {
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 100,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+          }
+        return request(app).get('/api/articles/1').expect(200)
+        .then(({body: {article}})=>{
+            expect(article).toMatchObject(article_1)
```

Comment:
I like how you've used the toMatchObject matcher here! Really importantly though, you also want to **make sure that the article on the response body has the newly generated article_id which matches the requested value** (here we'd expect article_id to be 1 for example). This is so we have some proof that the data is coming from our database!